workshops/templates/workshops/export_badges.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base_nav_fixed.html" %}
+
+{% block content %}
+<h3>Group by badge name</h3>
+<a href="{{ badges_api_link }}">API link.</a>
+<p>
+  {% if badges_json_link %}
```

Comment:
Isn't it always true? When I look into the view:
```python
badges_json_link = reverse('api:export-badges', kwargs={'format': 'json'})
```
When `reverse` can return something falsy?